packages/plugin-webpack/src/plugin.ts

```diff
@@ -1,12 +1,12 @@
 import { postProcessBundle, PostProcessOptions } from '@metamask/snap-utils';
-import { Compiler, WebpackPluginInstance } from 'webpack';
+import { Compiler } from 'webpack';
 import { RawSource } from 'webpack-sources';
 
 const PLUGIN_NAME = 'SnapsWebpackPlugin';
 
 export type Options = PostProcessOptions;
 
-export default class SnapsWebpackPlugin implements WebpackPluginInstance {
```

Comment:
How come we don't need this anymore? 🤔

Reply:
It breaks the type for whatever reason, and it seems like other plugins (including the ones by Webpack) don't do it either.